Description: change the font name from Monospace to GWMonospace 
 Change the font name from Monospace to GWMonospace in order to not override system's default monospace font
Author: Francesca Ciceri <madamezou@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/MonospaceBold.sfd
+++ b/MonospaceBold.sfd
@@ -1,7 +1,7 @@
 SplineFontDB: 1.0
-FontName: MonospaceBold
-FullName: Monospace Bold
-FamilyName: Monospace
+FontName: GWMonospaceBold
+FullName: GWMonospace Bold
+FamilyName: GWMonospace
 Weight: Bold
 Copyright: The programs to draw the splines of this font are Copyright 251 2000 by George Williams.
 Version: 001.000
--- a/MonospaceOblique.sfd
+++ b/MonospaceOblique.sfd
@@ -1,7 +1,7 @@
 SplineFontDB: 1.0
-FontName: MonospaceOblique
-FullName: Monospace Oblique
-FamilyName: Monospace
+FontName: GWMonospaceOblique
+FullName: GWMonospace Oblique
+FamilyName: GWMonospace
 Weight: Medium
 Copyright: The programs to draw the splines of this font are Copyright (c) 2000,2001 by George Williams.
 Version: 001.000
--- a/MonospaceRegular.sfd
+++ b/MonospaceRegular.sfd
@@ -1,7 +1,7 @@
 SplineFontDB: 1.0
-FontName: Monospace
-FullName: Monospace Regular
-FamilyName: Monospace
+FontName: GWMonospace
+FullName: GWMonospace Regular
+FamilyName: GWMonospace
 Weight: Medium
 Copyright: The programs to draw the splines of this font are Copyright (c) 2000,2001 by George Williams.
 Version: 001.000
